h/features/models.py

```diff
@@ -105,3 +112,57 @@ def remove_old_flags(cls):
 
     def __repr__(self):
         return '<Feature {f.name} everyone={f.everyone}>'.format(f=self)
+
+
+class Cohort(Base, mixins.Timestamps):
```

Comment:
So, thinking about this some more, I think this should be called `FeatureCohort`, the table should be called `featurecohort`, and the join table should be `featurecohort_user`.

The reason for this is that it's good for "ownership" of parts of the database to be clear. By ensuring all these tables start with `feature` it's clear they're owned by the `h.features` package.

It's worth saying that in general I'd shy away from names like `FeatureCohort` in Python because it "stutters" (e.g. `features.Cohort` is much nicer than `features.FeatureCohort`) but I think we maybe have to make an exception for models, which should match the table name. Or maybe not, maybe `Cohort` with a table name of `featurecohort` is fine? Thoughts? @chdorner? @seanh?